ssm/brokers.py

```diff
@@ -43,6 +43,8 @@ class StompBrokerGetter(object):
     def __init__(self, bdii_url):
         """Set up the LDAP connection and strings which are re-used."""
         # Set up the LDAP connection
+        logging.warning('LDAP is deprecated and will be removed in an upcoming version, ' +
+                        'migrate to AMS.')
         log.debug('Connecting to %s...', bdii_url)
```

Comment:
```suggestion
        logging.warning('LDAP is deprecated and will be removed in an upcoming version, '
                        'please set host locally in SSM config.')
        log.debug('Connecting to %s...', bdii_url)
```

Use implicit line continuation in parentheses rather than adding a + for concatenation.
Tweaked the message as it's not that we specifically want people to use AMS, just not the LDAP retrieval of the broker.